backend/models/JobPostingData.js

```diff
@@ -18,14 +18,17 @@ const JobPostingSchema = new Schema({
   matches: {
     type: Number,
   },
+  students: {
+    type: [String],
+  },
   dateCreated: {
     type: Date,
   },
   dateUpdated: {
     type: Date,
   },
-  matches: {
```

Comment:
why are we removing matches? we need this to display on the profile page I thought

Reply:
This is a duplicate field that he's removing